### ISSUE TITLE
2024-12-18, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#24]
# Today's Arxiv Papers

(1) [SafeAgentBench: A Benchmark for Safe Task Planning of Embodied LLM Agents](http://arxiv.org/abs/2412.13178v1), 2024-12-17

(2) [DnDScore: Decontextualization and Decomposition for Factuality Verification in Long-Form Text Generation](http://arxiv.org/abs/2412.13175v1), 2024-12-17

(3) [Algorithmic Fidelity of Large Language Models in Generating Synthetic German Public Opinions: A Case Study](http://arxiv.org/abs/2412.13169v1), 2024-12-17

(4) [C-FedRAG: A Confidential Federated Retrieval-Augmented Generation System](http://arxiv.org/abs/2412.13163v1), 2024-12-17

(5) [SWAN: Preprocessing SGD Enables Adam-Level Performance On LLM Training With Significant Memory Reduction](http://arxiv.org/abs/2412.13148v1), 2024-12-17

(6) [Are Your LLMs Capable of Stable Reasoning?](http://arxiv.org/abs/2412.13147v1), 2024-12-17

(7) [AI PERSONA: Towards Life-long Personalization of LLMs](http://arxiv.org/abs/2412.13103v1), 2024-12-17

(8) [AIR-Bench: Automated Heterogeneous Information Retrieval Benchmark](http://arxiv.org/abs/2412.13102v1), 2024-12-17

(9) [LMUnit: Fine-grained Evaluation with Natural Language Unit Tests](http://arxiv.org/abs/2412.13091v1), 2024-12-17

(10) [Modality-Inconsistent Continual Learning of Multimodal Large Language Models](http://arxiv.org/abs/2412.13050v1), 2024-12-17

(11) [OmniEval: An Omnidirectional and Automatic RAG Evaluation Benchmark in Financial Domain](http://arxiv.org/abs/2412.13018v1), 2024-12-17

(12) [The Emergence of Strategic Reasoning of Large Language Models](http://arxiv.org/abs/2412.13013v1), 2024-12-17

(13) [Unlocking LLMs: Addressing Scarce Data and Bias Challenges in Mental Health](http://arxiv.org/abs/2412.12981v1), 2024-12-17

(14) [Adaptations of AI models for querying the LandMatrix database in natural language](http://arxiv.org/abs/2412.12961v1), 2024-12-17

(15) [SnakModel: Lessons Learned from Training an Open Danish Large Language Model](http://arxiv.org/abs/2412.12956v1), 2024-12-17

(16) [FineGates: LLMs Finetuning with Compression using Stochastic Gates](http://arxiv.org/abs/2412.12951v1), 2024-12-17

(17) [Truthful Text Sanitization Guided by Inference Attacks](http://arxiv.org/abs/2412.12928v1), 2024-12-17

(18) [An Agentic Approach to Automatic Creation of P&ID Diagrams from Natural Language Descriptions](http://arxiv.org/abs/2412.12898v1), 2024-12-17

(19) [Question: How do Large Language Models perform on the Question Answering tasks? Answer:](http://arxiv.org/abs/2412.12893v1), 2024-12-17

(20) [ArtAug: Enhancing Text-to-Image Generation through Synthesis-Understanding Interaction](http://arxiv.org/abs/2412.12888v1), 2024-12-17

(21) [RAG-Star: Enhancing Deliberative Reasoning with Retrieval Augmented Verification and Refinement](http://arxiv.org/abs/2412.12881v1), 2024-12-17

(22) [Preference-Oriented Supervised Fine-Tuning: Favoring Target Model Over Aligned Large Language Models](http://arxiv.org/abs/2412.12865v1), 2024-12-17

(23) [Selective Shot Learning for Code Explanation](http://arxiv.org/abs/2412.12852v1), 2024-12-17

(24) [ClarityEthic: Explainable Moral Judgment Utilizing Contrastive Ethical Insights from Large Language Models](http://arxiv.org/abs/2412.12848v1), 2024-12-17

(25) [Benchmarking and Understanding Compositional Relational Reasoning of LLMs](http://arxiv.org/abs/2412.12841v1), 2024-12-17

(26) [From An LLM Swarm To A PDDL-Empowered HIVE: Planning Self-Executed Instructions In A Multi-Modal Jungle](http://arxiv.org/abs/2412.12839v1), 2024-12-17

(27) [FocusChat: Text-guided Long Video Understanding via Spatiotemporal Information Filtering](http://arxiv.org/abs/2412.12833v1), 2024-12-17

(28) [DSGram: Dynamic Weighting Sub-Metrics for Grammatical Error Correction in the Era of Large Language Models](http://arxiv.org/abs/2412.12832v1), 2024-12-17

(29) [Detecting Emotional Incongruity of Sarcasm by Commonsense Reasoning](http://arxiv.org/abs/2412.12808v1), 2024-12-17

(30) [Activating Distributed Visual Region within LLMs for Efficient and Effective Vision-Language Training and Inference](http://arxiv.org/abs/2412.12785v1), 2024-12-17

(31) [RemoteRAG: A Privacy-Preserving LLM Cloud RAG Service](http://arxiv.org/abs/2412.12775v1), 2024-12-17

(32) [A Survey on Sequential Recommendation](http://arxiv.org/abs/2412.12770v1), 2024-12-17

(33) [A Survey of Calibration Process for Black-Box LLMs](http://arxiv.org/abs/2412.12767v1), 2024-12-17

(34) [Using LLM-Generated Draft Replies to Support Human Experts in Responding to Stakeholder Inquiries in Maritime Industry: A Real-World Case Study of Industrial AI](http://arxiv.org/abs/2412.12732v1), 2024-12-17

(35) [ASAP: Advancing Semantic Alignment Promotes Multi-Modal Manipulation Detecting and Grounding](http://arxiv.org/abs/2412.12718v1), 2024-12-17

(36) [Enhancing Naturalness in LLM-Generated Utterances through Disfluency Insertion](http://arxiv.org/abs/2412.12710v1), 2024-12-17

(37) [More Tokens, Lower Precision: Towards the Optimal Token-Precision Trade-off in KV Cache Compression](http://arxiv.org/abs/2412.12706v1), 2024-12-17

(38) [Trigger$^3$: Refining Query Correction via Adaptive Model Selector](http://arxiv.org/abs/2412.12701v1), 2024-12-17

(39) [UniEntrezDB: Large-scale Gene Ontology Annotation Dataset and Evaluation Benchmarks with Unified Entrez Gene Identifiers](http://arxiv.org/abs/2412.12688v1), 2024-12-17

(40) [Uncertainty-Aware Hybrid Inference with On-Device Small and Remote Large Language Models](http://arxiv.org/abs/2412.12687v1), 2024-12-17

(41) [XTransplant: A Probe into the Upper Bound Performance of Multilingual Capability and Culture Adaptability in LLMs via Mutual Cross-lingual Feed-forward Transplantation](http://arxiv.org/abs/2412.12686v1), 2024-12-17

(42) [Detecting Document-level Paraphrased Machine Generated Content: Mimicking Human Writing Style and Involving Discourse Features](http://arxiv.org/abs/2412.12679v1), 2024-12-17

(43) [Train More Parameters But Mind Their Placement: Insights into Language Adaptation with PEFT](http://arxiv.org/abs/2412.12674v1), 2024-12-17

(44) [SEG-SAM: Semantic-Guided SAM for Unified Medical Image Segmentation](http://arxiv.org/abs/2412.12660v1), 2024-12-17

(45) [Predicting User Behavior in Smart Spaces with LLM-Enhanced Logs and Personalized Prompts (Data Description)](http://arxiv.org/abs/2412.12653v1), 2024-12-17

(46) [iPrOp: Interactive Prompt Optimization for Large Language Models with a Human in the Loop](http://arxiv.org/abs/2412.12644v1), 2024-12-17

(47) [LLM-based Discriminative Reasoning for Knowledge Graph Question Answering](http://arxiv.org/abs/2412.12643v1), 2024-12-17

(48) [Falcon: Faster and Parallel Inference of Large Language Models through Enhanced Semi-Autoregressive Drafting and Custom-Designed Decoding Tree](http://arxiv.org/abs/2412.12639v1), 2024-12-17

(49) [What External Knowledge is Preferred by LLMs? Characterizing and Exploring Chain of Evidence in Imperfect Context](http://arxiv.org/abs/2412.12632v1), 2024-12-17

(50) [Make Imagination Clearer! Stable Diffusion-based Visual Imagination for Multimodal Machine Translation](http://arxiv.org/abs/2412.12627v1), 2024-12-17

(51) [Jailbreaking? One Step Is Enough!](http://arxiv.org/abs/2412.12621v1), 2024-12-17

(52) [SynthCypher: A Fully Synthetic Data Generation Framework for Text-to-Cypher Querying in Knowledge Graphs](http://arxiv.org/abs/2412.12612v1), 2024-12-17

(53) [Gender Bias and Property Taxes](http://arxiv.org/abs/2412.12610v1), 2024-12-17

(54) [MultiLingPoT: Enhancing Mathematical Reasoning with Multilingual Program Fine-tuning](http://arxiv.org/abs/2412.12609v1), 2024-12-17

(55) [Don't Yell at Your Robot: Physical Correction as the Collaborative Interface for Language Model Powered Robots](http://arxiv.org/abs/2412.12602v1), 2024-12-17

(56) [LLMs are Also Effective Embedding Models: An In-depth Overview](http://arxiv.org/abs/2412.12591v1), 2024-12-17

(57) [Process-Supervised Reward Models for Clinical Note Generation: A Scalable Approach Guided by Domain Expertise](http://arxiv.org/abs/2412.12583v1), 2024-12-17

(58) [Understanding Emotional Body Expressions via Large Language Models](http://arxiv.org/abs/2412.12581v1), 2024-12-17

(59) [FCMR: Robust Evaluation of Financial Cross-Modal Multi-Hop Reasoning](http://arxiv.org/abs/2412.12567v1), 2024-12-17

(60) [Evaluating Zero-Shot Multilingual Aspect-Based Sentiment Analysis with Large Language Models](http://arxiv.org/abs/2412.12564v1), 2024-12-17

(61) [Task-Agnostic Language Model Watermarking via High Entropy Passthrough Layers](http://arxiv.org/abs/2412.12563v1), 2024-12-17

(62) [Seed-CTS: Unleashing the Power of Tree Search for Superior Performance in Competitive Coding Tasks](http://arxiv.org/abs/2412.12544v1), 2024-12-17

(63) [LLMCL-GEC: Advancing Grammatical Error Correction with LLM-Driven Curriculum Learning](http://arxiv.org/abs/2412.12541v1), 2024-12-17

(64) [When to Speak, When to Abstain: Contrastive Decoding with Abstention](http://arxiv.org/abs/2412.12527v1), 2024-12-17

(65) [Solid-SQL: Enhanced Schema-linking based In-context Learning for Robust Text-to-SQL](http://arxiv.org/abs/2412.12522v1), 2024-12-17

(66) [Generating Move Smart Contracts based on Concepts](http://arxiv.org/abs/2412.12513v1), 2024-12-17

(67) [Can Large Language Models Understand You Better? An MBTI Personality Detection Dataset Aligned with Population Traits](http://arxiv.org/abs/2412.12510v1), 2024-12-17

(68) [Can You Trust LLM Judgments? Reliability of LLM-as-a-Judge](http://arxiv.org/abs/2412.12509v1), 2024-12-17

(69) [Boosting LLM-based Relevance Modeling with Distribution-Aware Robust Learning](http://dx.doi.org/10.1145/3627673.3680052), 2024-12-17

(70) [Track the Answer: Extending TextVQA from Image to Video with Spatio-Temporal Clues](http://arxiv.org/abs/2412.12502v1), 2024-12-17

(71) [LinguaLIFT: An Effective Two-stage Instruction Tuning Framework for Low-Resource Language Tasks](http://arxiv.org/abs/2412.12499v1), 2024-12-17

(72) [NLSR: Neuron-Level Safety Realignment of Large Language Models Against Harmful Fine-Tuning](http://arxiv.org/abs/2412.12497v1), 2024-12-17

(73) [A Simple and Fast Way to Handle Semantic Errors in Transactions](http://arxiv.org/abs/2412.12493v1), 2024-12-17

(74) [A System for Microserving of LLMs](http://arxiv.org/abs/2412.12488v1), 2024-12-17

(75) [Boosting Long-Context Information Seeking via Query-Guided Activation Refilling](http://arxiv.org/abs/2412.12486v1), 2024-12-17

(76) [AutoSGNN: Automatic Propagation Mechanism Discovery for Spectral Graph Neural Networks](http://arxiv.org/abs/2412.12483v1), 2024-12-17

(77) [Human-in-the-Loop Generation of Adversarial Texts: A Case Study on Tibetan Script](http://arxiv.org/abs/2412.12478v1), 2024-12-17

(78) [RareAgents: Autonomous Multi-disciplinary Team for Rare Disease Diagnosis and Treatment](http://arxiv.org/abs/2412.12475v1), 2024-12-17

(79) [Knowledge Boundary of Large Language Models: A Survey](http://arxiv.org/abs/2412.12472v1), 2024-12-17

(80) [Core Context Aware Attention for Long Context Language Modeling](http://arxiv.org/abs/2412.12465v1), 2024-12-17

(81) [LLM is Knowledge Graph Reasoner: LLM's Intuition-aware Knowledge Graph Reasoning for Cold-start Sequential Recommendation](http://arxiv.org/abs/2412.12464v1), 2024-12-17

(82) [LITA: An Efficient LLM-assisted Iterative Topic Augmentation Framework](http://arxiv.org/abs/2412.12459v1), 2024-12-17

(83) [Graph Learning in the Era of LLMs: A Survey from the Perspective of Data, Models, and Tasks](http://arxiv.org/abs/2412.12456v1), 2024-12-17

(84) [PERC: Plan-As-Query Example Retrieval for Underrepresented Code Generation](http://arxiv.org/abs/2412.12447v1), 2024-12-17

(85) [Assessing the Limitations of Large Language Models in Clinical Fact Decomposition](http://arxiv.org/abs/2412.12422v1), 2024-12-17

(86) [Bridging the Gap: Enhancing LLM Performance for Low-Resource African Languages with New Benchmarks, Fine-Tuning, and Cultural Adjustments](http://arxiv.org/abs/2412.12417v1), 2024-12-16

(87) [Automated Generation of Massive Reasonable Empirical Theorems by Forward Reasoning Based on Strong Relevant Logics -- A Solution to the Problem of LLM Pre-training Data Exhaustion](http://arxiv.org/abs/2412.12408v1), 2024-12-16

(88) [Interpretable LLM-based Table Question Answering](http://arxiv.org/abs/2412.12386v1), 2024-12-16

(89) [LogBabylon: A Unified Framework for Cross-Log File Integration and Analysis](http://dx.doi.org/10.1145/3672608.3707883), 2024-12-16

(90) [How Different AI Chatbots Behave? Benchmarking Large Language Models in Behavioral Economics Games](http://arxiv.org/abs/2412.12362v1), 2024-12-16

(91) [Visual Instruction Tuning with 500x Fewer Parameters through Modality Linear Representation-Steering](http://arxiv.org/abs/2412.12359v1), 2024-12-16

(92) [BioRAGent: A Retrieval-Augmented Generation System for Showcasing Generative Query Expansion and Domain-Specific Search for Scientific Q&A](http://arxiv.org/abs/2412.12358v1), 2024-12-16

(93) [A Large Language Model Approach to Identify Flakiness in C++ Projects](http://arxiv.org/abs/2412.12340v1), 2024-12-16

(94) [Second Language (Arabic) Acquisition of LLMs via Progressive Vocabulary Expansion](http://arxiv.org/abs/2412.12310v1), 2024-12-16

(95) [Emergence of Abstractions: Concept Encoding and Decoding Mechanism for In-Context Learning in Transformers](http://arxiv.org/abs/2412.12276v1), 2024-12-16

(96) [SepLLM: Accelerate Large Language Models by Compressing One Segment into One Separator](http://arxiv.org/abs/2412.12094v1), 2024-12-16

(97) [Instruction-based Image Manipulation by Watching How Things Move](http://arxiv.org/abs/2412.12087v1), 2024-12-16

(98) [CPath-Omni: A Unified Multimodal Foundation Model for Patch and Whole Slide Image Analysis in Computational Pathology](http://arxiv.org/abs/2412.12077v1), 2024-12-16

(99) [CG-Bench: Clue-grounded Question Answering Benchmark for Long Video Understanding](http://arxiv.org/abs/2412.12075v1), 2024-12-16

(100) [Making FETCH! Happen: Finding Emergent Dog Whistles Through Common Habitats](http://arxiv.org/abs/2412.12072v1), 2024-12-16
